launch/node_component_process.launch.py

```diff
@@ -0,0 +1,62 @@
+import launch
+import launch.actions
+import launch.substitutions
+import launch_ros.actions
+import launch_ros.descriptions
+
+def generate_launch_description():
+    return launch.LaunchDescription([
+        # Nodes Definition
+        launch_ros.actions.Node(
+            package='turtlesim',
+            namespace='turtlesim1',
+            executable='turtlesim_node',
+            name='sim'
+        ),
+        launch_ros.actions.Node(
+            package='turtlesim',
+            namespace='turtlesim2',
+            executable='turtlesim_node',
+            name='sim'
+        ),
+        launch_ros.actions.Node(
+            package='turtlesim',
+            executable='mimic',
+            name='mimic',
+            remappings=[
+                ('/input/pose', '/turtlesim1/turtle1/pose'),
+                ('/output/cmd_vel', '/turtlesim2/turtle1/cmd_vel'),
+            ]
+        ),
+        launch.actions.DeclareLaunchArgument(
+            'node_prefix',
+            default_value=[launch.substitutions.EnvironmentVariable('USER'), '_'],
+            description='Prefix for node names'),
+        
+        # Components Definition
+        launch_ros.actions.ComposableNodeContainer(
```

Comment:
Would you like to add a descripiton about what is happening with the composable node? Or at least add a link to proper documentation?

Reply:
Done.